### DISCUSSION TITLE
中文字体加粗问题

### DISCUSSION THREAD

**Discussion by maokr** [jsvine/pdfplumber#1084] [Q&A]
PDF中的中文字体，无论其是否加粗，他们的fontname都是一致得，没有blod标志出现

**Comment by maokr** [comment]
![微信图片_20240124173547](https://github.com/jsvine/pdfplumber/assets/48988287/0569599e-9400-4aa1-94d5-da59d077a8f9)
![微信图片_20240124173609](https://github.com/jsvine/pdfplumber/assets/48988287/619ab6f6-fce5-4cdc-8325-69f46034f8eb)

**Comment by jsvine** [comment]
Hi @maokr, and thanks for your interest in this library. Can you provide the PDF? Without it, it will be very difficult for members of the community to provide suggestions.

As a general note: Fontnames in PDFs are fairly arbitrary; there is no guarantee (in this PDF or others) that bold text will be associated with a `fontname` with the phrase `Bold` in it.

**Reply by maokr** [reply]
Thank you for your reply,for example : page 1
![微信图片_20240125090054](https://github.com/jsvine/pdfplumber/assets/48988287/c564d0f7-c2a7-4636-8d07-48510578b397)
, it has chinese is bold and it also has chinese not bold, but i get fontname value,both are consistent
[格力地产2023年第三季度报告.pdf](https://github.com/jsvine/pdfplumber/files/14045430/2023.pdf)
for example

**Reply by jsvine** [reply]
Thank you for providing the PDF. Unfortunately, I also don't see any obvious characteristics that would help you distinguish between the text that looks bold and the text that does not.

**Comment by wind-chh** [comment]
There are several ways to make characters rendered like bold, using bold font is one of them.
